tests/ga/test_multi_config_extension.py

```diff
@@ -630,6 +630,70 @@ def test_it_should_handle_and_report_enable_errors_properly(self):
                 }
                 self._assert_extension_status(sc_handler, expected_extensions)
 
+    def test_it_should_handle_and_report_disallowed_extensions_properly(self):
```

Comment:
Could you also please add a case for multi config ext allowed by policy

Reply:
Added